deployment/src/strongmind_deployment/rails.py

```diff
@@ -82,10 +84,38 @@ def ecs(self):
         self.kwargs['env_vars'] = self.env_vars
         self.kwargs['container_image'] = container_image
 
-        self.container = ContainerComponent("container",
-                                            pulumi.ResourceOptions(parent=self),
-                                            **self.kwargs
-                                            )
+        web_entry_point = self.kwargs.get('web_entry_point', ["sh", "-c",
+                                                              "rails db:prepare db:migrate db:seed && "
+                                                              "rails assets:precompile && "
+                                                              "rails server --port 3000 -b 0.0.0.0"])
+
+        self.kwargs['entry_point'] = web_entry_point
+
+        self.web_container = ContainerComponent("container",
+                                                pulumi.ResourceOptions(parent=self),
+                                                **self.kwargs
+                                                )
+
+        self.need_worker = self.kwargs.get('need_worker', None)
+        if self.need_worker is None:
+            # If we don't know if we need a worker, check for sidekiq in the Gemfile
```

Comment:
lol. Nice.

Reply:
(this kind of stuff probably helps make it feel magical. I'm good with this).

Reply:
that said, I'm not sure I see tests for it? If people learned to depend on it ....